test/language/expressions/typeof/proxy.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2019 Ecma International. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: |
```

Comment:
for convention, please use `description: >`. The pipe preserves the line terminators.